cmd/mimir/config-descriptor.json

```
@@ -3288,6 +3288,17 @@
           "fieldType": "boolean",
           "fieldCategory": "experimental"
         },
+        {
+          "kind": "field",
+          "name": "ruler_max_results_per_rule",
+          "required": false,
+          "desc": "Maximum number of alerts/series each rule can generate. 0 to disable.",
```

```suggestion
          "desc": "Maximum number of alerts/series each rule can generate. 0 to disable.",
```
Does `/` mean "per", "or", or "and"?

---

It means "and", since the limit controls both how many alerts and how many series rules can generate (within Prometheus).